nrf_modem/include/nrf_socket.h

```diff
@@ -211,44 +188,60 @@ typedef uint32_t nrf_fd_set;
  * @ingroup nrf_socket
  * @{
  */
-#define NRF_SO_ERROR		       4
-#define NRF_SO_RCVTIMEO		       20
-#define NRF_SO_SNDTIMEO		       21
-#define NRF_SO_BINDTODEVICE	       25
-#define NRF_SO_SILENCE_ALL	       30
-/** Boolean control for ICMP echo reply disable/enable (0/1). Default is 1 (enabled). */
-#define NRF_SO_IP_ECHO_REPLY	       31
-/** Boolean control for ICMPv6 echo reply disable/enable (0/1). Default is 1 (enabled). */
-#define NRF_SO_IPV6_ECHO_REPLY	       32
-#define NRF_SO_REUSEADDR	       40
-/** Release Assistance Indication feature: This will indicate that the next call to send/sendto will
- * be the last one for some time, which means that the modem can get out of connected mode quicker
- * when this data is sent.
- */
-#define NRF_SO_RAI_LAST                    50
-/** Release Assistance Indication feature: This will indicate that the application will not send any
- * more data. This socket option will apply immediately, and does not require a call to send
- * afterwards.
- */
-#define NRF_SO_RAI_NO_DATA                 51
-/** Release Assistance Indication feature: This will indicate that after the next call to
- * send/sendto, the application is expecting to receive one more data packet before this socket
- * will not be used again for some time.
- */
-#define NRF_SO_RAI_ONE_RESP                52
-/** Release Assistance Indication feature: If a client application expects to use the socket more
- * it can indicate that by setting this socket option before the next send call which will keep
- * the modem in connected mode longer.
- */
-#define NRF_SO_RAI_ONGOING                 53
-/** Release Assistance Indication feature: If a server application expects to use the socket more
- * it can indicate that by setting this socket option before the next send call.
- */
-#define NRF_SO_RAI_WAIT_MORE               54
+/** Enable reuse of server addresses */
+#define NRF_SO_REUSEADDR 2
+/** Read and clear socket error status (read only). */
+#define NRF_SO_ERROR 4
+/** Receive timeout. */
+#define NRF_SO_RCVTIMEO 20
+/** Send timeout. */
+#define NRF_SO_SNDTIMEO 21
+/** Bind a socket to network interface identified by a Packet Data Network ID. */
+#define NRF_SO_BINDTODEVICE 25
+/** Disable ICMP echo replies on both IPv4 and IPv6.
+ *  Set to 1 to disable, or to 0 enable. Default is 0, disabled.
```

Comment:
```suggestion
 *  Set to 1 to disable, or to 0 to enable. Default is 0, disabled.
```

Reply:
First and second sentence conflicts. Is 0 enabled or disabled?

Reply:
Thanks, have clarified (I hope)